tangelo/linq/circuit.py

```diff
@@ -396,6 +412,18 @@ def simplify(self, max_cycles=100, param_threshold=1e-3, remove_qubits=False):
                                remove_qubits=remove_qubits)
         self.__dict__ = opt_circuit.__dict__
 
+    def controlled_measurement_op(self, measure):
```

Comment:
For these 2 functions: is it reasonable to expect the user to pass something that is not covered in these if branches ? If yes, how do we handle them ?

Currently, I think both the functions will do nothing and ignore what the user has passed, without any warning etc.